### DISCUSSION TITLE
How to build Prisma Client in Docker?

### DISCUSSION THREAD

**Discussion by kulterryan** [prisma/prisma#24995] [Q&A]
### Question

Hi There, 
I'm using Elysia and Bun with Prisma, when I'm trying to start the server, it is throwing the following error:

```
2024-08-11 16:50:09 backend start: PrismaClientInitializationError: Prisma Client could not locate the Query Engine for runtime "linux-arm64-openssl-1.1.x".
2024-08-11 16:50:09 backend start: 
2024-08-11 16:50:09 backend start: This happened because Prisma Client was generated for "darwin-arm64", but the actual deployment required "linux-arm64-openssl-1.1.x".
```

But I've added linux-arm64-openssl-1.1.x in the `binaryTargets` already:
```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["prismaSchemaFolder", "fullTextIndex", "fullTextSearch"]
  binaryTargets = ["native", "linux-arm64-openssl-1.1.x"]
}
```

### Error Code
<img width="942" alt="Screenshot 2024-08-11 at 4 49 13 PM" src="https://github.com/user-attachments/assets/c15da6d0-5685-4707-b7cc-d196ab44a5f5">

### Information about Prisma Schema, Client Queries and Environment (optional)

```
generator client {
  provider = "prisma-client-js"
  previewFeatures = ["prismaSchemaFolder", "fullTextIndex", "fullTextSearch"]
  binaryTargets = ["native", "linux-arm64-openssl-1.1.x"]
}
```

**Comment by ludralph** [comment]
Hi @kulterryan 👋 

Did you regenerate the Prisma Client after adding the recommended `binaryTarget`?

**Reply by kulterryan** [reply]
Yes, but that didn't worked. I managed to resolve the issue by simply generating the prisma client in root folder and exporting it from there!
